src/twisted/web/_flatten.py

```diff
@@ -178,7 +210,10 @@ def escapedComment(data: Union[bytes, str]) -> bytes:
     """
     if isinstance(data, str):
         data = data.encode("utf-8")
-    data = data.replace(b"--", b"- - ").replace(b">", b"&gt;")
+    if _hasMSOComments(data):
+        data = data.replace(b"--", b"- - ")
```

Comment:
```suggestion
    if _hasMSOComments(data):
        # Don't force escaping the closing tag angle bracket
        # for MS office, as we want them verbatim.
        data = data.replace(b"--", b"- - ")
```

Since I have not used this code, is hard to come with a good suggestion, but a comment would help.

Reply:
thank you for pointing that out, that's an oversight from my part, see [4810253](https://github.com/twisted/twisted/pull/11805/commits/481025321d7b95b19ed9213517e821a6a118128b)

I fixed it to be:
```
if not _hasMSOComments(data):
    data = data.replace(b"--", b"- - ").replace(b">", b"&gt;")
```